pyproject.toml

```diff
@@ -0,0 +1,55 @@
+[build-system]
+requires = ["setuptools>=64.0.1", "setuptools-scm[toml]>=6.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "libssa"
+dynamic = ["version"]
+authors = [
+    { name = "Kleydson Stenio" },
+]
+description = "Put a short project description here"
+readme = "README.md"
+license = { file = "LICENSE.txt" }
+requires-python = ">3.8,<3.11"
+classifiers = [
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "License :: OSI Approved :: GNU Affero General Public License v3 or later (AGPLv3+)",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+    "numpy~=1.25.1",
+    "scipy~=1.11.1",
+    "pandas~=2.0.3",
+    "scikit-learn~=1.3.0",
+    "PySide6~=6.4.3",
+    "pyqtgraph~=0.13.3",
+    "openpyxl~=3.1.2",
+    "psutil~=5.9.5",
+    "Markdown~=3.4.3"
```

Comment:
Generally, I think it's better to use `>=` and `<` for packaging and add a freezed `requirements.txt`, e.g., with pip-compile from pip-tools. That way the package can be easier used inside an environment with other tools.

Reply:
I never did this, so I'll have to check what you're meaning with pip-compile and freezed requirements. But I had some good experience using <= instead of ~= for requirements.txt with pip install.